chrome/content/zotero/tabs.js

```diff
@@ -244,6 +246,124 @@ var Zotero_Tabs = new function () {
 		tab.iconBackgroundImage = icon.style.backgroundImage;
 	};
 
+	this._createLabel = (title, filter) => {
+		let xhtmlNS = "http://www.w3.org/1999/xhtml";
+		let desc = document.createXULElement('description');
+		
+		let regex = new RegExp(`(${filter})`, 'gi');
+		let matches = title.matchAll(regex);
+
+		let lastIndex = 0;
+
+		for (let match of matches) {
+			if (match.index > lastIndex) {
+				// Add preceding text
+				desc.appendChild(document.createTextNode(title.substring(lastIndex, match.index)));
+			}
+			// Add matched text wrapped in <b>
+			let b = document.createElementNS(xhtmlNS, 'b');
+			b.textContent = match[0];
+			desc.appendChild(b);
+			lastIndex = match.index + match[0].length;
+		}
+
+		if (lastIndex < title.length) {
+			// Add remaining text
+			desc.appendChild(document.createTextNode(title.substring(lastIndex)));
+		}
+		return desc;
+	};
+
+	/**
+	 * Sets the data in the popup of opened tabs
+	 */
+	this._setOpenedTabsMenu = () => {
+		let openedTabsMenu = document.getElementById('zotero-opened-tabs-list');
+		// Empty existing nodes
+		while (openedTabsMenu.firstChild) {
+			openedTabsMenu.removeChild(openedTabsMenu.firstChild);
+		}
+		for (let tab of this._tabs) {
+			// Skip tabs whose title wasn't added yet
+			if (tab.title == "") {
+				continue;
+			}
+			if (!tab.title.toLowerCase().includes(this._openedTabsFilter)) {
+				continue;
+			}
+			// Top-level entry of the opened tabs array
+			let row = document.createXULElement('toolbaritem');
+			row.setAttribute('width', '400');
+			
+			// Cross button to close a tab
+			let closeButton = document.createXULElement('toolbarbutton');
+			closeButton.setAttribute('class', 'zotero-opened-tab-entry img-center close');
+
+			// Title of the opened tab
+			let tabName = document.createXULElement('toolbarbutton');
+			tabName.setAttribute('flex', '1');
+			tabName.setAttribute('class', 'zotero-opened-tab-entry title');
+			
+			let tabLabel = this._createLabel(tab.title, this._openedTabsFilter);
+			tabLabel.setAttribute('crop', 'end');
+			tabLabel.setAttribute('flex', 1);
+
+			tabName.appendChild(tabLabel);
+			
+			// Selected tab is bold
+			if (tab.id == this._selectedID) {
+				tabLabel.setAttribute('class', "header");
```

Comment:
Not sure where this class is from, but better to use `tabLabel.className = 'selected'` and add a specific CSS rule for this menu.